modules/index.js

```diff
@@ -0,0 +1,34 @@
+import BookList from './booklist';
+
+const book = new BookList();
+document.getElementById('form').addEventListener('submit', book.add);
+book.loadScreen();
+
+const dt = new Date();
+document.getElementById('date-time').innerHTML = dt;
+
+const list = document.getElementById('book-section');
+const addNew = document.getElementById('add-book');
+const contact = document.getElementById('contact-section');
+
+document.getElementById('list').addEventListener('click', () => {
+  if (!list.classList.contains('show')) {
+    list.classList.replace('hide', 'show');
+    addNew.classList.replace('show', 'hide');
+    contact.classList.replace('show', 'hide');
+  }
+});
+document.getElementById('add-b').addEventListener('click', () => {
+  if (!addNew.classList.contains('show')) {
+    list.classList.replace('show', 'hide');
+    addNew.classList.replace('hide', 'show');
+    contact.classList.replace('show', 'hide');
+  }
+});
+document.getElementById('contact-a').addEventListener('click', () => {
+  if (!contact.classList.contains('show')) {
+    list.classList.replace('show', 'hide');
+    addNew.classList.replace('show', 'hide');
+    contact.classList.replace('hide', 'show');
+  }
+});
```

Comment:
- [ ] Your app displays nothing and does not work and neither does its navigations. I am unable to see anything when I navigate. Kindly fix this.

  Homepage | Add book section | Contact section
  :---:|:---:|:---:
  ![dangelo-1](https://user-images.githubusercontent.com/36542986/159345574-07e96119-9b26-4ba4-b34c-c29d33273d61.jpg) | ![dangelo-2](https://user-images.githubusercontent.com/36542986/159345578-42d1831b-3b68-4ce9-b936-9a4634120cc2.jpg) | ![dangelo-3](https://user-images.githubusercontent.com/36542986/159345580-d60000e9-3492-4149-8d75-89f80852693b.jpg)